modules/api/src/main/java/net/countercraft/movecraft/TrackedLocation.java

```diff
@@ -0,0 +1,57 @@
+package net.countercraft.movecraft;
+
+import org.bukkit.World;
+import org.jetbrains.annotations.NotNull;
+
+public class TrackedLocation {
+    private int posX;
+    private int posY;
+    private int posZ;
```

Comment:
These should be stored as a `MovecraftLocation` and called the `offset`, since they are not an absolute position in the world but an offset from the `origin`.